File: client/src/app/_dtos/admin/callRecordItemToAddDto.ts
import { ICallRecordItem } from "src/app/_models/admin/callRecordItem";

export interface ICallRecordItemToAddDto
{
    personId: string;
    personType: string;
    categoryRef: string;
    adviseByMail: boolean;
    adviseBySMS: boolean;
    callRecordItem: ICallRecordItem;
}

File: api/DTOs/Admin/CallRecordItemToAddDto.cs
using api.Entities.Admin;

namespace api.DTOs.Admin
{
    public class CallRecordItemToAddDto
    {
        public string PersonId { get; set; }
        public string PersonType { get; set; }
        public string CategoryRef { get; set; }
        public string CustomerCity { get; set; }
        public bool AdviseByMail { get; set; }
        public bool AdviseBySMS { get; set; }
        public CallRecordItem callRecordItem { get; set; }
    }
}